### DISCUSSION TITLE
can't open video writer

### DISCUSSION THREAD

**Discussion by zhbUp** [bluenviron/mediamtx#1197] [Questions]
i start a rtsp-simple-server first, then, i use cv2.VideoWriter to write per frame data to this server,  it keep prompting "can't open video writer"

how should I handle it please ?

**Comment by aler9** [comment]
Hello, as written in the README, check that OpenCV has been compiled with GStreamer support:

```
python3 -c 'import cv2; print(cv2.getBuildInformation())'
```

Check that the output contains GStreamer: YES.

Otherwise you have to re-compile OpenCV by following instructions here:

https://github.com/aler9/rtsp-simple-server#from-opencv

[a comment by an automated account is not shown]
